kubernetes-client/src/main/resources/META-INF/services/io.fabric8.kubernetes.client.extension.ExtensionAdapter

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
?

Reply:
That looks unintentional.  I'll back that out as this file is going away in the other pr.